File: 15-11-2023 - Diagramas de flujo/sueldo.js
let horasTrabajadas, sueldo;
const costoHora = 30;
horasTrabajadas = parseInt(prompt('Ingrese la cantidad de horas trabajadas por el empleado: '));
/* costoHora = parseFloat(prompt('Ingrese el costo de la hora')); */

sueldo = horasTrabajadas * costoHora;

document.write(`El sueldo del empleado es: U$S ${sueldo}`);

File: 15-11-2023 - Diagramas de flujo/sueldo.go
package main

import "fmt"

func main() {
	var horasTrabajadas, costoHora, sueldo int

	fmt.Print("Ingrese la cantidad de horas trabajadas: ")
	fmt.Scan(&horasTrabajadas)
	fmt.Print("Ingrese el costo por hora: ")
	fmt.Scan(&costoHora)

	sueldo = horasTrabajadas * costoHora

	fmt.Print("El sueldo del empleado es: ", sueldo)
}
